scripts/lib/libvirt/compute-config

```diff
@@ -32,6 +32,8 @@ def main():
                         help="Virtual Disk Directory (e.g /dev/cloud)")
     parser.add_argument("bootorder", type=str,
                         help="Boot Order (e.g. 2)")
+    parser.add_argument("numcontrollers", type=str,
+                        help="Number of controller nodes")
```

Comment:
You may be right here, but other arguments in this function are set as strings although they are used as ints. I kept the same style.